transaction/interfaces.py

```diff
@@ -22,12 +22,19 @@ class ITransactionManager(Interface):
     """
 
     def begin():
-        """Begin a new transaction.
+        """Explicitly begin a new transaction.
 
         If an existing transaction is in progress, it will be aborted.
 
-        The newTransaction() method of registered synchronizers is called,
+        The ``newTransaction`` method of registered synchronizers is called,
         passing the new transaction object.
+
+        Note that transactions may be started implicitly without
+        calling ``begin``. In that case, ``newTransaction`` isn't
```

Comment:
I suspect that if we thought of them as one and the same and treated them as such this would be a lot less confusing :) (But maybe that's just because that's what I used to do in my own closed-source Java transaction package.)